myapp/playlist.js

```diff
@@ -0,0 +1,67 @@
+var songJSON=JSON.parse(playlistsJSON);
```

Comment:
Please use clear and well defined names for vars. It makes reviewing thinks very hard, because you cannot follow execution flow. Eg in this situation you can use: parsedPlaylists etc as name